lib/ember-test-helpers/test-module.js

```diff
@@ -230,6 +230,13 @@ export default Klass.extend({
     this.container = items.container;
     this.registry = items.registry;
 
+
+    var viewRegistry = this.container.lookup('-view-registry:main');
+    if (viewRegistry) {
+      var View = Ember.__loader.require('ember-views/views/view').default;
+      View.views = viewRegistry;
```

Comment:
Yeah, I agree that this should be fixed upstream in Ember (we should inject `-view-registry:main` into both `view` and `component`), but the solution with the smallest amount of change might just be:

```javascript
module.component = Ember.Component.create({
        layout: template,
        container: module.container,
        _viewRegistry: module.container.lookup('-view-registry:main')
      });
``` 

We should still fix that injection upstream though....